src/styles/main.scss

```diff
@@ -12,3 +12,22 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

Comment:
All good, but why did you write it here. You already have `src/styles/_catalog.scss`. You should write it here

Reply:
Maybe I misunderstood this rule)
![1](https://user-images.githubusercontent.com/98978706/162180160-7baf557b-7437-4af3-925d-9cfb2f403f55.PNG)
But I fixed files